cucumber.gemspec

```diff
@@ -18,34 +18,34 @@ Gem::Specification.new do |s|
   }
 
   # Keep in sync with .circleci/config.yml & .rubocop.yml
-  s.required_ruby_version = '>= 2.3'
-  s.add_dependency 'builder', '>= 2.1.2'
-  s.add_dependency 'cucumber-core', '~> 4.0'
-  s.add_dependency 'cucumber-expressions', '~> 7.0.0'
-  s.add_dependency 'cucumber-formatter-dots', '~> 1.0'
-  s.add_dependency 'cucumber-wire', '~> 1.0'
-  s.add_dependency 'diff-lcs', '~> 1.3'
-  s.add_dependency 'gherkin', '~> 6.0'
-  s.add_dependency 'multi_json', '>= 1.7.5', '< 2.0'
-  s.add_dependency 'multi_test', '>= 0.1.2'
+  s.required_ruby_version = '>= 2.4'
+  s.add_dependency 'builder', '~> 3.2', '>= 3.2.3'
+  s.add_dependency 'cucumber-core', '~> 5.0', '>= 5.0.0'
+  s.add_dependency 'cucumber-expressions', '~> 8.0', '>= 8.0.0'
+  s.add_dependency 'cucumber-formatter-dots', '~> 1.2', '>= 1.2.0'
+  s.add_dependency 'cucumber-wire', '~> 1.0', '>= 1.0.0'
+  s.add_dependency 'diff-lcs', '~> 1.3', '>= 1.3'
+  s.add_dependency 'gherkin', '~> 7.0', '>= 7.0.3'
+  s.add_dependency 'multi_json', '~> 1.13', '>= 1.13.1'
+  s.add_dependency 'multi_test', '~> 0.1', '>= 0.1.2'
 
-  s.add_development_dependency 'aruba', '~> 0.14'
-  s.add_development_dependency 'json', '~> 1.8.6'
-  s.add_development_dependency 'nokogiri', '~> 1.8.1'
-  s.add_development_dependency 'pry'
-  s.add_development_dependency 'rake', '>= 12.0'
-  s.add_development_dependency 'rspec', '>= 3.6'
-  s.add_development_dependency 'rubocop', '~> 0.64.0'
-  s.add_development_dependency 'simplecov', '>= 0.6.2'
-  s.add_development_dependency 'syntax', '>= 1.0.0'
+  s.add_development_dependency 'aruba', '~> 0.14', '>= 0.14.11'
+  s.add_development_dependency 'json', '~> 2.2', '>= 2.2.0'
+  s.add_development_dependency 'nokogiri', '~> 1.10', '>= 1.10.4'
+  s.add_development_dependency 'pry', '~> 0.12', '>= 0.12.2'
+  s.add_development_dependency 'rake', '~> 12.3', '>= 12.3.3'
+  s.add_development_dependency 'rspec', '~> 3.8', '>= 3.8.0'
+  s.add_development_dependency 'rubocop', '~> 0.74', '>= 0.74.0'
+  s.add_development_dependency 'simplecov', '~> 0.17', '>= 0.17.0'
+  s.add_development_dependency 'syntax', '~> 1.2', '>= 1.2.2'
 
   # For maintainer scripts
-  s.add_development_dependency 'octokit'
+  s.add_development_dependency 'octokit', '~> 4.14', '>= 4.14.0'
 
   # Needed for examples (rake examples)
-  s.add_development_dependency 'capybara', '>= 2.1'
-  s.add_development_dependency 'rack-test', '>= 0.6.1'
-  s.add_development_dependency 'sinatra', '>= 1.3.2'
+  s.add_development_dependency 'capybara', '~> 3.28', '>= 3.28.0'
```

Comment:
this is a sizeable bump. Are we sure we're happy bumping everything immediately to their latest versions

Reply:
Yes we are, as long as it doesn't break the build.

It looks like none of the `/examples` run in CI. Nor can I run them locally. I'll see if I can make them work. If not we might as well remove the capybara dependency